src/lib/integration_test_cloud_engine/coda_automation.ml

```diff
@@ -76,7 +76,7 @@ module Network_config = struct
     in
     let testnet_name = "integration-test-" ^ test_name in
     (* HARD CODED NETWORK VALUES *)
-    let coda_automation_location = "../automation" in
+    let coda_automation_location = "/home/steck/tmp/coda-automation" in
```

Comment:
I should find a better way to organize the path to coda_automation repo. I also changed this in my own PR.

Reply:
yes, sorry, that was meant to be a temp local change ... will fix

Reply:
I will work on threading engine specific cli arguments up to the test_executive to fix this issue.